core/src/main/java/org/apache/iceberg/actions/BaseExpireSnapshotsActionResult.java

```diff
@@ -22,13 +22,19 @@
 public class BaseExpireSnapshotsActionResult implements ExpireSnapshots.Result {
 
   private final long deletedDataFilesCount;
+  private final long deletedPosDeleteFilesCount;
+  private final long deletedEqDeleteFilesCount;
   private final long deletedManifestsCount;
   private final long deletedManifestListsCount;
 
   public BaseExpireSnapshotsActionResult(long deletedDataFilesCount,
+                                         long deletedPosDeleteFilesCount,
```

Comment:
It's also used in Spark 2.4 code https://github.com/apache/iceberg/blob/master/spark/v2.4/spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java#L274

Reply:
We don't have to implement the same functionality for older versions in this PR.
Maybe, just keep the old constructor for older Spark versions?

Reply:
Kept old constructor for older Spark versions (deleteFileCounts initialized to 0)

Reply:
Kept old constructor for older Spark versions (deleteFileCounts initialized to 0)